include/records.hrl

```diff
@@ -32,6 +32,11 @@
           auth_plugin_data :: binary()
          }).
 
+%% used under caching_sha2_password
+-record(sha2_auth_read, {
+    msg :: integer()
```

Comment:
It is possible to say that the type is `3|4`. It is a more exact type than `integer()`.

We could also decode this number to atoms such as `ok | full_auth` instead of 3 and 4. when we are parsing the packet. (It is just a suggestion, not very important.)